RELEASE.md

```diff
@@ -58,6 +58,17 @@ The release note are incorrectly generated, and needs to be manually regenerated
 This requires editing the release notes in the GitHub UI, and selecting the correct previous tag.
 Resolution of this issue is tracked in [Auto generation of release notes failure](https://github.com/Kuadrant/kuadrant-operator/issues/1211).
 
+## Creating the RC2+
+
+1. Create a local branch based off the, targeted release branch. 
+Example: `git checkout -B release-v1.2.0-rc2 release-v1.2`
+2. Cherry pick the changes required to fix which ever issue justified the creation of a new RC.
+Example: `git cerry-pick <commit sha>`
```

Comment:
```suggestion
Example: `git cherry-pick <commit sha>`
```

Reply:
@Boomatang ok to make this change? Then can approve

Reply:
Sorry, I though pushed that.